ivy/container/sorting.py

```diff
@@ -48,6 +128,52 @@ def argsort(
         *,
         out: Optional[ivy.Container] = None
     ) -> ivy.Container:
+        """
+        ivy.Container instance method variant of ivy.argsort. This method simply wraps the
```

Comment:
this line is too long, please update it so that one line only has up to 88 characters, rest of the content would go to a new line.